data/mods/Aftershock/spells/enchantments.json

```
@@ -25,8 +25,55 @@
   },
   {
     "type": "enchantment",
-    "id": "protect_cold",
+    "id": "forcefield_bash_cut_weak",
     "condition": "ACTIVE",
-    "values": [ { "value": "ARMOR_COLD", "add": -15 } ]
+    "values": [ { "value": "ARMOR_BASH", "add": -5 }, { "value": "ARMOR_CUT", "add": -5 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_bash_cut_medium",
+    "condition": "ACTIVE",
+    "values": [ { "value": "ARMOR_BASH", "add": -10 }, { "value": "ARMOR_CUT", "add": -10 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_bash_cut_heavy",
+    "condition": "ACTIVE",
+    "values": [ { "value": "ARMOR_BASH", "add": -15 }, { "value": "ARMOR_CUT", "add": -15 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_ballistic_piercing_weak",
+    "condition": "ACTIVE",
+    "hit_me_effect": [
+      {
+        "id": "conversion_field_flash",
+        "hit_self": true,
+        "message": "Your shield reacts with a flash of light and sound.",
+        "npc_message": "%1$s's shield goes off with a flash of light and sound."
+      }
+    ],
+    "values": [ { "value": "ARMOR_BULLET", "add": -10 }, { "value": "ARMOR_STAB", "add": -10 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "forcefield_ballistic_piercing_medium",
+    "condition": "ACTIVE",
+    "intermittent_activation": [
+      {
+        "id": "static_field",
+        "hit_self": true,
+        "message": "Your shield reacts with an arc of electricity.",
+        "npc_message": "%1$s's shield goes off with an arc of electricity."
+      }
+    ],
```

I think this is what you intended
```suggestion
    "hit_me_effect": [
      {
        "id": "static_field",
        "hit_self": true,
        "message": "Your shield reacts with an arc of electricity.",
        "npc_message": "%1$s's shield goes off with an arc of electricity."
      }
    ],
```